package.json

```diff
@@ -34,6 +37,7 @@
     "babel-loader": "^7.1.0",
     "babel-preset-env": "^1.5.2",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
```

Comment:
Ah that's right. In gallery, I enabled this feature by depending on `babel-plugin-transform-object-rest-spread` and adding `transform-object-rest-spread` to `.babelrc`.

For this project, we're making use of `babel-preset-env`, which gives you an easy way to add plugins. You can remove the `babel-preset-stage-2` dependency, and instead update `.babelrc`:

```
{
  "presets": [
    "react",
    [
      "env", {
        "include": [
          "babel-plugin-transform-class-properties",
          "babel-plugin-transform-flow-strip-types",
          "babel-plugin-transform-object-rest-spread"
        ],
      }
    ]
  ]
}
```

You'll have to add these dependencies to `package.json`:

```
"babel-plugin-transform-class-properties": "^6.24.0",
"babel-plugin-transform-flow-strip-types": "^6.22.0",
"babel-plugin-transform-object-rest-spread": "^6.23.0",
```

Also, unrelated to this, but please add the `styled-components` plugin to package.json`

```
"babel-plugin-styled-components": "^1.1.7",
```